tests/scripts/run_profiler_regressions.sh

```diff
@@ -5,22 +5,24 @@ source scripts/tools_setup_common.sh
 set -eo pipefail
 
 run_additional_T3000_test(){
-    remove_default_log_locations
-    mkdir -p $PROFILER_ARTIFACTS_DIR
+    if [ "$ARCH_NAME" != "grayskull" ]; then
```

Comment:
Going forward please do `if grayskull return` in such cases so the rest of the code does not have to be that indentended.

Reply:
Would this not be better if we have this as "ARCH_NAME" = wormhole_b0?
This script would trigger if we introduce another ARCH in the future

Reply:
Okay I will modify it to trigger only for wormhole_b0 instead of skipping it for GS

Reply:
can you also update the same reference in this file too?
I believe there is another reference doing the same thing  for run_async_mode_T3000_test

Reply:
Great idea, please make sure. 
https://github.com/tenstorrent/tt-metal/actions/workflows/t3000-profiler-tests.yaml after the change. 

https://github.com/tenstorrent/tt-metal/actions/workflows/run-profiler-regression.yaml should also pass as part of the post commit change.